packages/cozy-procedures/src/components/Documents.jsx

```diff
@@ -25,12 +30,20 @@ class Documents extends React.Component {
           <LoadingDocumentHolder />
           <DocumentHolder />
         </div>
-        {fields.map((document, index) => (
-          <section key={index}>
-            <Label>{t(`documents.labels.${document.label}`)}</Label>
-            <EmptyDocumentHolder />
-          </section>
-        ))}
+        {Object.values(populatedTemplateDocsWithStore).map(
+          (documentTemplate, index) => {
+            const { documents: documentsFromStore } = documentTemplate
+            return (
+              <section key={index}>
+                <Label>{t(`documents.labels.${documentTemplate.label}`)}</Label>
+                <DocumentsGroups
+                  documents={documentsFromStore}
+                  templateDoc={documentTemplate}
```

Comment:
I know we talked about this structure together, but seeing it now, I feel like it was maybe misguided. We call `mergeDocsFromStoreAndTemplate` to merge the 2 data structures, in then we split them again a few lines later :/

Reply:
In fact, now I just need templateDoc for the `count` attribute. I'll edit that :)